v63003/u_inref/gtm8767.csh

```diff
@@ -0,0 +1,35 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+set randint = $$
```

Comment:
As discussed, this is not a random number and should be replaced with a genrandnumbers.csh call.
Additionally, I would like a hard_spin_count of 0 and 2147483647 also tested as being allowed and a value of 2147483648 tested as being not allowed.